src/index.js

```diff
@@ -1,51 +1,27 @@
-require ('./linagora.esn.profile/app/show/follower/profile-show-follower.controller.js');
-require ('./linagora.esn.profile/app/show/follower/subheader/profile-show-follower-subheader.component.js');
-require ('./linagora.esn.profile/app/show/following/profile-show-following.component.js');
-require ('./linagora.esn.profile/app/show/following/profile-show-following.controller.js');
-require ('./linagora.esn.profile/app/show/following/subheader/profile-show-following-subheader.component.js');
-require ('./linagora.esn.profile/app/show/overview/profile-overview.component.js');
-require ('./linagora.esn.profile/app/show/overview/profile-overview.controller.js');
-require ('./linagora.esn.profile/app/show/profile-show.component.js');
-require ('./linagora.esn.profile/app/show/tabs/profile-tabs.component.js');
-require ('./linagora.esn.profile/app/show/tabs/profile-tabs.controller.js');
-require ('./linagora.esn.profile/app/show/timeline/profile-show-timeline.component.js');
-require ('./linagora.esn.profile/app/show/timeline/subheader/profile-show-timeline-subheader.component.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/esn.router.js');
+require('esn-frontend-common-libs/src/frontend/js/constants.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/header/header.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/user/user.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/sidebar.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/module-registry/module-registry.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/user-configuration/user-configuration.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/business-hours/business-hours.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/config/config.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/i18n/i18n.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/home-page/home-page.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/application-menu.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/core.js');
+require('esn-frontend-common-libs/src/frontend/components/dynamic-directive/src/dynamic-directive.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/http.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/session.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/profile.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/notification.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/timeline/timeline.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/previous-page.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/async-action.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/availability/availability.module.js');
```

Comment:
I do not agree with this. The "esn-frontend-common-libs" dependencies should be required in the modules definition files, which are `./linagora.esn.controlcenter/app/app.module.js` and `./linagora.esn.profile/app/app.module.js`.

The index.js file can then be:

```javascript
require('esn-frontend-common-libs/src/index.js');
require('./require-angular-injections.js');

require('./linagora.esn.controlcenter/app/app.module.js');
require('./linagora.esn.profile/app/app.module.js');
```
The issue you had in your chat comment "when  removing local dependencies from index js and put it in app.module.js  , the application crash" was most likely due to you requiring a module directive|service|controller before the module definition, like for example:

```javascript
// DON'T DO THAT
require ('./linagora.esn.controlcenter/app/app.route.js'); // module doesn't exist yet and we try to add something inside
require('./linagora.esn.controlcenter/app/app.module.js'); // module is created here
```
Order matters, and webpack cannot order it automatically.